docs-www/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@brainhubeu/gatsby-docs-kit-start",
   "description": "Gatsby-Docs-Kit starter repository",
-  "version": "0.0.6",
+  "version": "1.10.18",
```

Comment:
this line https://github.com/brainhubeu/react-carousel/blob/master/docs-www/package.json#L18 should be modified instead

Reply:
Whoops, sorry 😅
Fixed.